screens/checkout.js

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable no-unused-vars */
+import { useTailwind } from "tailwind-rn";
+import {
+  View,
+  Text,
+  Button,
+  TouchableOpacity,
+  FlatList,
+  useWindowDimensions,
+  StyleSheet,
+} from "react-native";
+import { Image } from "react-native-web";
+import { faker } from "@faker-js/faker";
+import { ListViewCard } from "../components/cards";
+import { IconInput } from "../components/inputs";
+import { RoundButton } from "../components/buttons";
+
+let ORDER = [];
+Array.from({ length: 3 }).forEach((_, index) => {
+  ORDER.push({
+    title: faker.company.name(),
+    rating: (Math.random() * (5 - 1) + 1).toFixed(1),
+    image: faker.image.food(_, _, true),
+    id: index,
+
+    foodPrice: (Math.floor(Math.random() * 10) + 9.99).toFixed(2),
+    serviceFee: (Math.floor(Math.random() * 10) + 1).toFixed(2),
+    deliveryFee: (Math.floor(Math.random() * 10) + 1).toFixed(2),
+    deliveryTime: `${Math.floor(Math.random() * 5) + 1}-${
+      Math.floor(Math.random() * 15) + 5
+    } min`,
+  });
+});
+
+const getBreakPoint = (width) => {
```

Comment:
We are using this function in a few places now. Let's create a utils directory at the root of the project and move this function to a file called `utils/screen.js`. Then export it from `screen.js` and import it to this file. Additionally, let's also replace the function in the other locations its being used.